kubernetes/cmsweb/services/reqmon-tasks.yaml

```diff
@@ -38,7 +38,7 @@ spec:
         name: reqmon-tasks
         resources:
           requests:
-            memory: "256Mi"
+            memory: "5Gi"
```

Comment:
Alan, how we jup from 256MI to 5Gi limit? This limited defines how much RAM required to start the up. Does reqmon and reqmon-tasks really need 5GI to start?

Reply:
This is one of the issues we spotted this year (8GB of memory footprint).
The DataCache thread contains too much data and 5GB was the average of memory footprint a couple of weeks back when I logged into the cmsweb backends and checked the service's memory consumption.